app/models/grda_warehouse/config.rb

```diff
@@ -39,6 +39,15 @@ def self.available_cas_flag_methods
       }
     end
 
+    def self.available_verified_homeless_history_method
+      {
+        'Include enrollments from data sources that are visible in the window.' => :visible_in_window,
+        'Include enrollments that are visible to the user.' => :visible_to_user,
+        'Include all enrollments if client has a release that grants the user access to the client. If no release, only include enrollments that are visible to the user.' => :release,
+        'Include all enrollments' => :all_enrollments,
+      }
+    end
+
```

Comment:
I can't comment on it here, but line 283 below needs `:verified_homeless_history_method,` or this won't save.